Wireless.py

```diff
@@ -127,12 +127,12 @@ def transmit(self, sender, frame):
 
     # ======================== private =========================================
 
-    def _computePdrPisterHack(self, sender_pos, receiver_pos):
+    def _computePdrPisterHack(self, sender, receiver):
```

Comment:
can you please rename to `_computerPdr()`; the fact that it's Pister-Hack is inside the black box